pode.build.ps1

```diff
@@ -105,22 +129,16 @@ function Invoke-PodeBuildDotnetBuild($target) {
 #>
 
 # Synopsis: Stamps the version onto the Module
-task StampVersion {
+Task StampVersion {
     (Get-Content ./pkg/Pode.psd1) | ForEach-Object { $_ -replace '\$version\$', $Version } | Set-Content ./pkg/Pode.psd1
     (Get-Content ./pkg/Pode.Internal.psd1) | ForEach-Object { $_ -replace '\$version\$', $Version } | Set-Content ./pkg/Pode.Internal.psd1
-    (Get-Content ./packers/choco/pode.nuspec) | ForEach-Object { $_ -replace '\$version\$', $Version } | Set-Content ./packers/choco/pode.nuspec
-    (Get-Content ./packers/choco/tools/ChocolateyInstall.ps1) | ForEach-Object { $_ -replace '\$version\$', $Version } | Set-Content ./packers/choco/tools/ChocolateyInstall.ps1
+    (Get-Content ./packers/choco/pode_template.nuspec) | ForEach-Object { $_ -replace '\$version\$', $Version } | Set-Content ./packers/choco/pode.nuspec
+    (Get-Content ./packers/choco/tools/ChocolateyInstall_template.ps1) | ForEach-Object { $_ -replace '\$version\$', $Version } | Set-Content ./packers/choco/tools/ChocolateyInstall.ps1
 }
 
 # Synopsis: Generating a Checksum of the Zip
-task PrintChecksum {
-    if (Test-PodeBuildIsWindows) {
-        $Script:Checksum = (checksum -t sha256 $Version-Binaries.zip)
-    }
-    else {
-        $Script:Checksum = (shasum -a 256 ./$Version-Binaries.zip | awk '{ print $1 }').ToUpper()
-    }
-
+Task PrintChecksum {
+    $Script:Checksum =(Get-FileHash ./deliverable/2.10.0-Binaries.zip -Algorithm SHA256).Hash
```

Comment:
You just reminded me about this, haha! The hardcoded `2.10.0` here should be `$Version` instead :D

Reply:
Opppppssss